internal/commands/root.go

```diff
@@ -94,8 +94,8 @@ type credentialOptions struct {
 }
 
 func (o *credentialOptions) addFlags(flags *pflag.FlagSet) {
-	flags.StringVar(&o.targetContext, "target-context", "", "Context on which the application is executed")
-	flags.StringArrayVarP(&o.credentialsets, "credential-set", "c", []string{}, "Use a credentialset (either a YAML file, or a credential set present in the credential store)")
+	flags.StringVar(&o.targetContext, "target-context", "", "Context on which the application is installed")
```

Comment:
Should we indicate that default value is `<current-active-context>` and can also be set using env var `DOCKER_TARGET_CONTEXT`?

Reply:
I will add the default but won't add the `DOCKER_TARGET_CONTEXT` as we don't do that for things like `DOCKER_HOST`. It should be documented though.